example/l2fwd_simple/l2fwd_simple_run.sh

```diff
@@ -29,4 +29,17 @@ fi
 
 rm -f pcapout.pcap
 
+./odp_l2fwd_simple${EXEEXT} null:0 null:1 \
+	02:00:00:00:00:01 02:00:00:00:00:02 &
+
+sleep 1
+kill -s SIGINT $!
+wait $!
+STATUS=$?
+
+if [ "$STATUS" -ne 255 ]; then
+  echo "Error: status was: $STATUS, expected 255"
+  exit 1
```

Comment:
just exit $STATUS. This would allow underlying program to return 42 to mark the test as skipped

Reply:
In theory, a STATUS 0 would be considered an error here because the expected status is -1 which is converted to 255.